staging/src/k8s.io/code-generator/cmd/validation-gen/validators/format.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2024 The Kubernetes Authors.
```

Comment:
nit: I believe for new file Copyright should be 2025

Reply:
The code itself is 2024, and that's what the copyright applies to.

Reply:
Ah sorry, makes sense thanks.  Resolving